experimental/iterators/include/iterators/Utils/RuntimeUtils.h

```diff
@@ -23,7 +23,13 @@
 //===---------------------------------------------------------------------===//
 // Exposed C API.
 //===---------------------------------------------------------------------===//
-extern "C" RUNTIME_UTILS_EXPORT int64_t oneWay();
-extern "C" RUNTIME_UTILS_EXPORT void otherWay(int64_t);
+
+extern "C" RUNTIME_UTILS_EXPORT int8_t *makeSampleInputOperator();
+
+// XXX(ingomueller): resolve name clash with namespace
```

Comment:
Please do it now, we don't want to carry this "disambiguation by naming" forward.

Reply:
Fixed with #344 and latest commit, which also uses "poor-man's namespaces" (prefixing the C-functions with `iterators*`).